cypress/e2e/CMS/$Article/index.js

```diff
@@ -164,28 +171,62 @@ And('public use text is not displayed', () => {
     cy.get("div[class='public-use']").should('not.exist');
 });
 
+And('user selects {string} content item', (dropDown) => {
+    cy.get(`input[value='${dropDown}']`).click({ force: true });
+});
 
+And('user clicks on {string} link in the {string} text area', (title, cartOption) => {
+    cy.get(`div.paragraph-type-title:contains('${cartOption}')`).parent().parent().find(`span:contains('${title}')`).parent().click();
+});
 
+And('user clicks on {string} button item', (title) => {
+    cy.get(`input[value="${title}"]`).click({ force: true })
+});
 
+And('user selects {string} item from main page content', (title) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find(`td:contains("${title}")`).parent().find('input').check();
+});
 
+Then('the promo image is matching the earlier selected image', () => {
+    function extractImgName(imageFullName) {
```

Comment:
helper function should never be a part of a step - you should put it in utils folder and import where needed

Reply:
done